roles/openshift_manageiq/tasks/main.yaml

```diff
@@ -1,4 +1,8 @@
 ---
+- fail:
+    msg: "use_manageiq requires OpenShift Enterprise 3.1 or Origin 1.1."
+  when: openshift.common.version_greater_than_3_1_or_1_1 | bool == False
```

Comment:
@detiber, does this look better?  I'm not entirely certain I have the syntax correct.